tests/processors/base/normalizers/test_phase_normalizer.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from processors.base.normalizers import get_normalized_phase
+import pytest
```

Comment:
The usual importing order we follow is:

1. Standard libraries
2. Third-party libraries
3. Our modules